backend/src/main/validation/PromotionValidation.ts

```diff
@@ -44,6 +44,9 @@ export class PromotionValidation {
       .default(new Date()),
     /** note incoming format is a string, and Joi will automatically convert to type Date */
     expirationDate: Joi.date().required(),
+    lat: Joi.number().required().strict(true), // note incoming format is a string, and Joi will automatically convert to type Number (both lat/lon)
```

Comment:
`// note incoming format is a string, and Joi will automatically convert to type Number (both lat/lon)`
I think this comment is not needed anymore